src/bokeh/embed/bundle.py

```diff
@@ -44,10 +45,9 @@
 from ..core.templates import CSS_RESOURCES, JS_RESOURCES
 from ..document.document import Document
 from ..model import Model
-from ..resources import BaseResources, Resources
+from ..resources import Resources
 from ..settings import settings
```

Comment:
Is this import unused? I searched the entire file changes and I didn't see `settings` referenced anywhere

Reply:
`settings.dev` is used in `_bundle_extensions()`.